charts/influxdb/README.md

```diff
@@ -98,12 +98,28 @@ The command removes all the Kubernetes components associated with the chart and
 | config.tls | [Details](https://docs.influxdata.com/influxdb/v1.7/administration/config/#tls) | {} |
 | initScripts.enabled | Boolean flag to enable and disable initscripts. If the container finds any files with the extensions .sh or .iql inside of the /docker-entrypoint-initdb.d folder, it will execute them. The order they are executed in is determined by the shell. This is usually alphabetical order. | false |
 | initScripts.scripts | Init scripts | {} |
-| backup.enabled | Enable backups, if `true` must configure one of the storage providers | `false` |
-| backup.gcs | Google Cloud Storage config | `nil`
-| backup.azure | Azure Blob Storage config | `nil`
-| backup.schedule | Schedule to run jobs in cron format | `0 0 * * *` |
-| backup.annotations | Annotations for backup cronjob | {} |
-| backup.podAnnotations | Annotations for backup cronjob pods | {} |
+| `backup.enabled`                                  | enable InfluxDB backup                                                                                                                                                                    | `false`                                                 |
+| `backup.directory`                                | directory where backups are stored in                                                                                                                                                     | `"/backups"`                                            |
+| `backup.retentionDays`                            | retention time in days for backups (older backups are deleted)                                                                                                                            | `10`                                                    |
```

Comment:
Please specify that this is for PVC and not the long term object store.

Do you expect some users to just keep backups in a PVC and not in an object store?

Reply:
some people may not even have an object store available.
also the retention works on the object store by syncing this dir.

Reply:
the object store retention should be managed independently such as via https://cloud.google.com/storage/docs/lifecycle

Reply:
hence the request to specify it for clarity

Reply:
done